framework/include/base/Assembly.h

```diff
@@ -434,10 +438,29 @@ class Assembly
   const Node * const & nodeNeighbor() const { return _current_neighbor_node; }
 
   /**
-   * Creates the volume, face and arbitrary qrules based on the orders passed in.
+   * Creates volume, face and arbitrary qrules based on the orders passed in
+   * that apply to all subdomains.
    */
   void createQRules(QuadratureType type, Order order, Order volume_order, Order face_order);
 
+  /**
+   * Creates block-specific volume, face and arbitrary qrules based on the
+   * orders passed in.  Any quadrature rules specified using this function
+   * override those created via in the non-block-specific/global createQRules
+   * function.
+   */
```

Comment:
Need documenting the parameters. It is not so clear to me the difference of the three order parameters.

Reply:
Yeah - I was just being lazy here since the original createQRules function didn't say anything about them either.  But really - only the `order` parameter is unclear I think.  I'll push up a clarification.

Reply:
yes.